File: tasks/smallestMultiple.java
int smallestMultiple(int left, int right) {

  for (int candidate = 1;; candidate++) {
    boolean correct = true;
    for (int divisor = left; divisor <= right; divisor++) {
      if (candidate % divisor != 0) {
        correct = false;
        break;
      }
    }
    if (correct) {
      return candidate;
    }
  }
}

File: tasks/smallestMultiple.js
function smallestMultiple(left, right) {

  for (var candidate = 1;; candidate++) {
    var correct = true;
    for (var divisor = left; divisor <= right; divisor++) {
      if (candidate % divisor) {
        correct = false;
        break;
      }
    }
    if (correct) {
      return candidate;
    }
  }
}
